modules/special-rules/pages/barrage.adoc

```diff
@@ -4,29 +4,56 @@ These weapons can saturate a wide area of the battlefield with intense salvoes o
+* Cover as many units in the target detachment as possible.
+. Count the number of units of the target detachment that are under the template.
+This number is the Firepower value that each {barrage} unit contributes to the total Firepower that your detachment directs at the target detachment.
 
-+[TODO: It'd be good to rationalise this rule a bit, given that (1) infantry units don't always have five individual models on them, and (2) some people like to put their vehicles on bases, and (3) it isn't clear how much of a vehicle or war engine should count to resolve disagreement when a template barely 'nicks' such a unit.]+
+[IMPORTANT]
 ====
-. Count the number of units beneath the template.
-The number of units under the template is your unit's Firepower value for this shot.
+When you place Blast markers, {barrage} weapons contribute to your detachment's total Firepower, not the number of super-heavy weapons.
 
-. Add this Firepower value to the rest of the Firepower of your detachment for the current attack. 
-
-IMPORTANT: For the placement of Blast markers, count a {barrage} shot along with regular Firepower and not as a super-heavy weapon.
+For suppression by Blast markers, each {barrage} weapon is a single super-heavy weapon, so can be suppressed by a single Blast marker.
+====
 
 .A classic {barrage} unit
 ====
 The Space Marine Whirlwind armoured vehicle is armed with a Whirlwind missile launcher.
 What this vehicle lacks in range it can make up for in speed and the ability to rain a destructive barrage of missiles down onto an area of the battlefield.
-====
\ No newline at end of file
+====
+
+.Adding up {barrage} Firepower
+====
+You shoot three Space Marine Whirlwind units at an Ork Warband detachment that is partially intermingled with a Kult of Speed detachment.
+
+.Adding up {barrage} Firepower. Numbered rectangles are Ork Boyz units; solid circles are Gunwagon units; semi-opaque circles are Barrage templates. Green units are in the Ork Warband detachment; red units are in the Kult of Speed detachment.
+image::barrage-example-1a.png[One Barrage template that covers Ork Infantry and Vehicle units,320]
+
+You place one Barrage template for all three Whirlwinds.
+
+In total, the template covers three units from the target detachment:
+
+* The template covers Ork Boyz (1) and (2), and the Gunwagon.
+* Ork Boyz (3) is not covered by the template because their centre is not under the template.
+* We ignore Ork Boyz (4) as it is not in the target detachment.
+// TODO: But we don't ignore it completely do we? We'll resolve a separate attack on that detachment at 3 Firepower, won't we?
+
+So, each Whirlwind adds 3 Firepower to the total for your detachment.
+As the Whirlwinds have no other help, the total is 9 Firepower against the Ork Warband detachment.
+====
+
+// TODO IJW: decide how to determine if detachments are intermingled, and if it's needed.
```

Comment:
I don't think we need this now do we? Line #17 seems to cover it:

>* Cover as many units in the target detachment as possible.

Reply:
I think this relates to your comment on the PR...?

> Barrages - do we want to add in the results of the FB discussion about intermingled enemy detachments, or go back and add that later?

Reply:
Correct, the example needs to be updated to say that FP3 is directed against the other detachment.

Sorry for being a bit brief...

Reply:
No worries. Righto, I'll add a suggestion to cover this in the example soon. We have a conversation prompt for that a bit further up, so I'll remove your TODO note with a suggestion here, and then resolve this conversation.
```suggestion
```